critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -23,8 +23,10 @@
     </div>
     <div class="form-group">
       <small class="text-muted pull-left" style="font-style:italic;">
-        {{ _('You can use <a href="%(url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
-             url='http://daringfireball.net/projects/markdown/syntax') }}
+          {{ _('You can read the <a href="%(style_guide_url)s" target="_blank">style guide</a> for writing good CritiqueBrainz reviews!
+                Remember, you can use <a href="%(markdown_url)s" target="_blank">Markdown</a> syntax to apply custom formatting.',
```

Comment:
Also, I suggest we change `Remember` to  `Also` or `Furthermore`.
Also, the next sentence (about using `markdown`) gets little out of focus. We can make use of a `<br/>` bringing it to a new line.

Reply:
Amended.